src/logic/map_objects/tribes/ship.h

```diff
@@ -325,10 +372,17 @@ struct Ship : Bob {
 		Serial worker_economy_serial_{kInvalidSerial};
 		uint32_t destination_{0U};
 		uint32_t capacity_{0U};
+		uint32_t warship_soldier_capacity_{0U};
```

Comment:
why do we have different data types in the loader here in comparison to the defin tions above e.g. line 357?